src/repl/session.rs

```diff
@@ -287,7 +291,8 @@ impl Session {
                     true,
                     Some("Deployment".into()),
                 ) {
-                    Ok((_, result)) => {
+                    Ok((ref mut res_output, result)) => {
+                        output_err.append(res_output);
```

Comment:
On `Ok(_)`, I think we should be appending in `output`, instead of `output_err`?

Reply:
Following up on the extra log that we would like to remove in clarinet, I think we could move the snippet L:545 in this file

```rust
if let Some((ref contract_name, _, _, _, _)) = result.contract {
    let snippet = format!("→ .{} contract successfully stored. Use (contract-call? ...) for invoking the public functions:", contract_name.clone());
    output.push(green!(snippet));
}
```

to line 522:
```rust
let mut result = match self.formatted_interpretation(
    snippet.to_string(),
    None,
    true,
    None,
) {
    Ok((mut output, result)) => {
        if let Some((ref contract_name, _, _, _, _)) = result.contract {
            let snippet = format!("→ .{} contract successfully stored. Use (contract-call? ...) for invoking the public functions:", contract_name.clone());
            output.push(green!(snippet));
        }
        output
    },
    Err(result) => result,
};
```

So that the extra line is only added when developers are interacting with the contracts from the console.

Reply:
If we append to output, then the warnings are suppressed in addition to that "contract successfully stored" message. The problem is that both the warnings and that message are together coming out of `formatted_interpretation`. There is only one `Vet<String>` returned from that method, because before, there was only diagnostics if it failed. Should I just add another buffer to that tuple being returned, so we have an output and an err_output coming out of `formatted_interpretation`?